utils/crypto/bls/public.go

```diff
@@ -9,7 +9,10 @@ import (
 	blst "github.com/supranational/blst/bindings/go"
 )
 
-const PublicKeyLen = blst.BLST_P1_COMPRESS_BYTES
+const (
+	SecretKeyLen = blst.BLST_SCALAR_BYTES
```

Comment:
It feels weird to me to export the `SecretKeyLen` from a package that doesn't handle secret keys anymore (because the length of the key is going to be representation specific).

Would it make more sense to export this from the `localsigner` package?